content/articles/introduction-to-udoo-x86-ii-ultra/index.md

```diff
@@ -0,0 +1,106 @@
+
+#### Efficiency
+Udoo X86 II Ultra is capable of running software meant for a normal personal computer. This is made possible by a 2.56GHz quad-core intel CPU and an 8GB RAM.
+
+#### Availability of freely useful guides and community
+Through a helpful community even not all guides are available online.
+
+###	Lower energy consumptions
+The Arduino Leonardo 101 board requires less than 11 Watts of energy which makes it very energy efficient.
+
+#### Large storage capacity
+Udoo X86 II Ultra box comes with a 32GB embedded multimedia card. However, users might decide to upgrade the storage to fit their desire. Though expensive, an SSD card is usually the best option. Additionally, a micro-SD card can also be used as a storage media.
+
+#### X86-compatible architecture
+Udoo X86 II Ultra supports most operating systems including Linux, Windows and Android. For Linux and Android, the internal storage can be used as a read only memory with additional SATA `HDD/SSD/eMMC` drive used as /tmp and /home directories.
+
+Udoo II X86 Ultra is also used a gaming emulator as it supports most retro gaming emulators available for a PC. It can also emulate new consoles like play-station II.
+
+#### Decent GPIO support
+The Udoo x86 Ultra comes with 28 `GPIO` ports in total which may seem few, but considering that it has an Arduino 101 embedded on it, this is a decent number.
+
+The Arduino board has an in built functionality for what some pins can be used for (like a six-axis accelerometer and gyroscope) and it adds 12 additional GPIO pins. Several key communication protocols such as the two IIC, two UART, LPC and SDIO are also supported.
+
+####  Easy and straightforward set up
+It’s absolutely straightforward to set up. The installation guidelines for the supported operating systems are provided in hard-copy and video form.
+
+#### Arduino compatible
+Udoo x86 II Ultra comes with an embedded Arduino 101, with all upgrades enabled (including 6-axis accelerometer, gyroscope and Bluetooth).
+
+#### Great media streaming potential
+Udoo x86 II Ultra has a moderately high power that makes it a single-board computer for streaming media in a high quality. It is able to stream a 30Hz 4k video on at least three monitors through two mini display ports.
+
+It also supports a set up for UHD capable media station with a wide variety of applications like Spotify, Netflix, YouTube, Kodi etc.
+
+### Cons
+Udoo X86 II Ultra might not the best choice for the following reasons:
+
+#### Expensive
+Udoo x86 II Ultra is relatively expensive compared to other session board controllers on the market.
+
+#### Lack of official documentation
+Apart from the hardware specifications sheet, there is no official documentation publicly available. Also, there is generally not too many project examples or information available on the official website for Udoo x86 II ultra at the moment. Therefore, users have a hard time finding out how to use the board.
```

Comment:
Provide a link to the official site mentioned above.

Reply:
Already addressed.